src/components/NewWorkspaceModal.js

```diff
@@ -99,18 +99,25 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
       onSuccess(await Utils.cond(
         [cloneWorkspace, async () => {
           const workspace = await Ajax().Workspaces.workspace(cloneWorkspace.workspace.namespace, cloneWorkspace.workspace.name).clone(body)
+          if (isAzureBillingProject() && !getConfig().isProd) {
+            Ajax().Apps.createAppV2(`wds-${workspace.workspaceId}`, workspace.workspaceId)
+          }
           const featuredList = await Ajax().FirecloudBucket.getFeaturedWorkspaces()
           Ajax().Metrics.captureEvent(Events.workspaceClone, {
             featured: _.some({ namespace: cloneWorkspace.workspace.namespace, name: cloneWorkspace.workspace.name }, featuredList),
             ...extractCrossWorkspaceDetails(cloneWorkspace, {
               // Clone response does not include cloudPlatform, cross-cloud cloning is not supported.
-              workspace: _.merge(workspace, { cloudPlatform: cloneWorkspace.workspace.cloudPlatform })
+              workspace: _.merge(workspace, { cloudPlatform: getProjectCloudPlatform() })
```

Comment:
I know there's been some back-and-forth on this code … as it stands, does this line actually need to change? This seems unrelated to the auto-deploy code.

Reply:
It does not need to change, but there is also not harm in changing it.

Reply:
I've kept it as is with the proposed code -- happy to change back too though!

Reply:
Check if the case of billing project cloud platform matches that expected by workspaces. Workspace endpoints return `Gcp` and `Azure` while billing projects return `GCP` and `AZURE`.

Reply:
That's a good point, but for the purpose of the event emitting it doesn't matter-- I'm always upper-casing the cloudPlatform for consistency.

https://github.com/DataBiosphere/terra-ui/blob/3718f12d3ef210d2cf1b0935992c71cb8efb1f90/src/libs/events.js#L123

Reply:
Agreed the case doesn't matter for metrics, as long as it's consistent.